src/index.ts

```diff
@@ -16,7 +16,8 @@
  */
 
 import { RPReporter } from './reporter';
+import { ReportingApi } from './reportingApi';
+import { LOG_LEVELS, STATUSES } from './constants';
```

Comment:
Is it supposed to be used somewhere out of the agent?

Reply:
LOG_LEVELS, STATUSES will be used for other ReportingApi methods, but now they are not needed and I will remove them